license_header.txt

```diff
@@ -1,5 +1,5 @@
 
-Copyright 2022 Red Hat, Inc.
+Copyright 2022-2023 Red Hat, Inc.
```

Comment:
should we leave it as the year `2023`?  This is supposed to be for new files that we create.  Existing files missing the headers should have the year they were created (we'd have to go back and fix them)